### ISSUE TITLE
Does an admin create a new User ?

### ISSUE THREAD

**Issue by ankitjain28may** [ncs-jss/plexus-17#103]

**Comment by itaditya** [comment]
Yes, for testing purposes an admin is allowed to create user. This feature is added of line 63 in https://github.com/ncs-jss/plexus-17/blob/master/routes/user.route.js

**Comment by ankitjain28may** [comment]
So, there is a bug in the code, as we are only checking whether the authId is there or not.. but when admin creates a new user... he can add only email and name so a new user is created but.. after creating a single user... admin wont be able to create more users as per the condition as the user with authId = undefined is already there.

**Comment by itaditya** [comment]
Yeah, I knew about that, thanks for bringing this up. Can you suggest a possible patch for this.

**Comment by itaditya** [comment]
Also, create an issue regarding the same
